multicluster/charts/linkerd-multicluster/README.md

```diff
@@ -87,6 +88,7 @@ Kubernetes: `>=1.21.0-0`
 | gateway.serviceAnnotations | object | `{}` | Annotations to add to the gateway service |
 | gateway.serviceType | string | `"LoadBalancer"` | Service Type of gateway Service |
 | gateway.terminationGracePeriodSeconds | string | `""` | Set terminationGracePeriodSeconds on gateway deployment |
+| gateway.tolerations | object | `{}` | Tolerations for the gateway pod |
```

Comment:
shouldn't this be an array rather than an object?

Reply:
Fixed it.